rita_common/src/usage_tracker/mod.rs

```diff
@@ -84,19 +84,27 @@ pub struct FormattedPaymentTx {
 }
 
 fn to_formatted_payment_tx(input: PaymentTx) -> FormattedPaymentTx {
-    match input.txid {
-        Some(txid) => FormattedPaymentTx {
+    if let Some(txid) = input.txid {
+        FormattedPaymentTx {
             to: input.to,
             from: input.from,
             amount: input.amount,
             txid: format!("{txid:#066x}"),
-        },
-        None => FormattedPaymentTx {
+        }
+    } else if let Some(tx_hash) = input.tx_hash {
```

Comment:
why was this change needed?

Reply:
In b21 pmt struct can either be txid from eth or tx_hash from althea, guess i missed this change during the adding althea wallets pr